_layouts/question-include.html

```diff
@@ -8,7 +8,7 @@
     {% else %}
         This is not a cake nor a cookie
     {% endcase %}
-    <a href="#back" id="backButton" class="usa-button usa-button--outline">Back</a> <a href="#next" id="nextButton" class="usa-button">Next</a>
+    <a href="#back" id="backButton" class="usa-button usa-button--outline margin-top-3">Back</a> <a href="#next" id="nextButton" class="usa-button">Next</a>
```

Comment:
I did some other stuff with placement of buttons, so we probably don't need this.